scripts/build-tools.js

```diff
@@ -14,11 +23,28 @@ const buildTools = async () => {
       resolve(__dirname, '../config', 'tools-automated.json'),
       JSON.stringify(automatedTools, null, '  ')
     );
-    await combineTools(automatedTools, manualTools);
+    await combineAutomatedAndManualTools(automatedTools);
   } catch (err) {
     console.log(err);
     throw err
   }
 };
 
-buildTools();
\ No newline at end of file
+const buildToolsManual = async () => {
+  try {
+    const automatedTools = require('../config/tools-automated.json');
+    await combineAutomatedAndManualTools(automatedTools);
+  } catch (err) {
+    console.log(err);
+    throw err;
+  }
+};
+
+if (require.main === module) {
```

Comment:
did you add it because of the script trigger from package.json? if yes, please make it clear with a code comment, for future generations 😆 and me, cause I'm old and keep forgetting things 😆

Reply:
You caught me! I left a comment there, now if our brains decide to take a day off, it's got our back! 😆